server/persistence/persistence_test.go

```diff
@@ -39,6 +40,17 @@ func TestInMemoryDatabase(t *testing.T) {
 	db := persistence.NewInMemoryDB()
 	testDb(t, db)
 }
+func TestTelemetryTable(t *testing.T) {
+
+	db := persistence.NewInMemoryDB()
+	telemetry.SetMetric(db.Instance, telemetry.DeployStatus, "SUCCESS")
+	telemetry.SetMetric(db.Instance, telemetry.AccessType, "PRIVATE")
+	retrieved := telemetry.Telemetry{}
+	retrieved = telemetry.Metric(db.Instance, telemetry.DeployStatus)
```

Comment:
If you like you could add setting an already set metric (and verifying it was updated) and getting a non-existent metric to the test..

Reply:
Discussed this, maybe skip the first of those but add the second.

Reply:
Added